R/status_addin.R

```diff
@@ -0,0 +1,98 @@
+#' Addin for viewing status of orderly develop mode.
+#'
+#' Allows users to reload dependencies and to cleanup development mode.
+#'
+#' @export
+status_addin <- function() {
```

Comment:
Looking at this I think some sort of colour coding will be needed to make the information pop out